src/callbacks/onLayout/customizePlotStyleToggle.js

```diff
@@ -0,0 +1,14 @@
+import stopAnimation from './initStudyDayControl/stopAnimation';
+
+export default function customizePlotStyleToggle() {
+    const chart = this;
+    this.controls.wrap
+        .selectAll('.control-group')
+        .filter(d => d.option === 'plot_max_values')
+        .selectAll('input')
+        .on('change', function(d) {
+            chart.config.plot_max_values = d;
+            stopAnimation.call(chart);
```

Comment:
This workflow makes sense to me ... but it still isn't working when you change this control during the animation. I had trouble using stopAnimation like this before ... will play around with it a bit.

Reply:
Relates to #271